archai/nlp/datasets/exp_utils.py

```diff
@@ -194,8 +194,8 @@ def setup_dllogger(enabled=True, filename=os.devnull):
                 ),
             ]
         dllogger.init(backends)
-    else:
-        dllogger.init([])
+    # else:
```

Comment:
Could you comment why?

Reply:
During search with training, we run a single script that initializes several models throughout the search, using the same `init()` function that we have in `train.py`. When `dllogger` is initialized more than once in the same script, it throws an error saying that it has already been initialized, even though no `backends` are being passed (it happens because we are calling its `init()` function).

Since the `dllogger` is not implemented by us, we need to make a workaround that allow multiple initializations. And this has been the least "invasive" way I found to do it.